src/flowcean/transforms/flatten.py

```diff
@@ -79,7 +75,7 @@ def transform(self, data: pl.DataFrame) -> pl.DataFrame:
             data = data.with_columns(
                 [
                     pl.col(feature)
-                    .list.eval(pl.first().struct.field("value"))
```

Comment:
same here?

Reply:
Same as above, a leftover from debugging. Polars recommends using `pl.element()` to get the current element in an `eval` expression. However, `pl.first()` works just as well - I just wanted to see if that's the breaking bug.